_posts/2015-11-18-Stats-2015.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  "2015"
+author: orta
+categories: stats community 
+---
+
+> CocoaPods is a dependency manager only in the sense that Apple is a cell phone manufacturer – it’s just one of their many projects.
+
+--  Ash Furrow - [Building Online Communities](http://ashfurrow.com/blog/building-online-communities/)
+
+Last year, around this time, I started an [annual write-up](/Stats-2014/) for CocoaPods. A lot happens in a project with 10 relatively active developers, and a lot of drive-by contributors. 
+
+<!-- more -->
+
+## Contributions
+
+- Boris / Marius / Segiddins heading up
+- Segiddin's contrbutions from Capital One
+- Perhaps get number of contributors to main repo?
+
+## Web Properties
+
+- CocoaPods.org
+  - QI
+  - Pod Pages
+  - Owner Pages
+  - Search Traffic Doubled
+- Trunk Stats?
+- Overview of web properties?
+- CocoaDocs getting Button $$$
+- CocoaDocs Jazzy Support
+
+- More Windows users than iOS
+- Most accessed search queries are menu/calendar
+
+## Community
+
+- Google / Amazon / Facebook / Twitter
```

Comment:
Don't forget Apple. First party adoption in ResearchKit.

https://github.com/ResearchKit/ResearchKit/pull/5